#2596 The selection of images in the binary graphic component is not shown to the user

Patrykb0802 opened the pull request:
- Added highlightSelectedElement function that checks which image has been selected  (zero/one state) and turning on the background highlight for one that is selected and removing highlight on previously selected one.

- Used that function in createImageList function inside loop, which creates list of available images.

Patrykb0802 committed:
#2596 The selection of images in the binary graphic component is not shown to the user

- Added highlightSelectedElement function that checks which image has been selected  (zero/one state) and turning on the background highlight for one that is selected and removing highlight on previously selected one.

- Used that function in createImageList function inside loop, which creates list of available images.

Limraj reviewed (commented):
1. When editing a component, the selected graphic is visible.
2. There is still a problem with marking the graphics that were selected after refreshing the page and editing the component.
3. This can be improved in a much simpler way, just fix the functions: setZeroImage and setOneImage. Correcting these functions will fix the situation where we edit and select graphics, and read previously selected graphics. I am proposing a fix:
[#2596_The_selection_of_images_in_the_binary_graphic_component_is_not_shown_to_the_user.patch](https://github.com/SCADA-LTS/Scada-LTS/files/12316712/2596_The_selection_of_images_in_the_binary_graphic_component_is_not_shown_to_the_user.patch)
There is a logic of selecting graphics in these functions, but the selected method of selecting the object does not work, the upper border is not visible:
`$("graphicRendererBinaryImageSetZero"+ graphicRendererEditor.zeroImage).border = "2";
`

Patrykb0802 committed:
#2596 The selection of images in the binary graphic component is not shown to the user

- Changed feedback display method to show gray background of chosen image instead of border that was not visible in functions setZeroImage and setOneImage

Patrykb0802 committed:
#2596 The selection of images in the binary graphic component is not shown to the user

- Changed feedback display method to show gray background of chosen image instead of border that was not visible in functions setZeroImage and setOneImage
